Src/FluentAssertions.DataSets/FluentAssertions.DataSets.csproj

```diff
@@ -59,25 +51,9 @@
       <PackagePath>contentFiles\any\any\</PackagePath>
```

Comment:
Yes, because this `.cs` is added to the project that uses this package and will allow this package to add the specific implementations of `IEquivalencyStep` needed for working with `DataSet`.

Reply:
Isn't that done in FA core via assembly scanning?
I've tried removing this inclusion of `InitializeDataSetSupport.cs` in the nuget package, creating a nuget package of this project and then referencing that nuget package from a separate solution.
From that other solution I could successfully run `When_data_sets_are_identical_equivalence_test_should_succeed`.

Reply:
1. This package will inject `InitializeDataSetSupport.cs` to the test project that adds a `[AssertionEngineInitializer]` to the test project that ensures `DataSet` support is configured correctly. 
1. Before the first assertion happens, the new assembly scanning feature introduced in v7 searches for the `[AssertionEngineInitializer]` and causes the 

Without this, it all depends on when the `FluentAssertions.DataSets` package is loaded. With all the lazy-loading .NET is doing, this results in a very unpredictable behavior.

Reply:
Roger 👍 
So because `GetAssemblies` only loads at _currently loaded_ assemblies, we might miss FluentAssertions.DataSet.
https://github.com/fluentassertions/fluentassertions/blob/aff3f7829056083c5280f7c0f530df5645d83dc0/Src/FluentAssertions/Common/Services.cs#L83-L88

Then how about adding `<Visible>false</Visible>` to hide it?

In cases of transitive inclusion, e.g. `TestProject -> MiddleProject -> FluentAssertions.DataSet`, do you know if this adds `InitializeDataSetSupport` to `TestProject` and/or `MiddleProject`?

Reply:
> So because `GetAssemblies` only loads at _currently loaded_ assemblies, we might miss FluentAssertions.DataSet.

Exactly. And `GetReferences` only gets the assemblies that are needed by the code currently loaded. 

> Then how about adding <Visible>false</Visible> to hide it?

What do you mean? To hide it from the solution explorer when opening `FluentAssertions.DataSets`

> In cases of transitive inclusion, e.g. TestProject -> MiddleProject -> FluentAssertions.DataSet, do you know if this adds InitializeDataSetSupport to TestProject and/or MiddleProject?

It'll be added to `MiddleProject`